### PR TITLE
ci: reduce output when running tests

### PR DESCRIPTION
**Pull request by charlierudolph**
cleans up the final noisy output

Before: https://travis-ci.org/Originate/exosphere/jobs/242142678
After: https://travis-ci.org/Originate/exosphere/jobs/242143958

### CODE REVIEW

File: exo-sync/features/steps/steps-given.ls [hunk 1]
```diff
@@ -26,5 +26,4 @@ defineSupportCode ({Given}) ->
     # create a new commit in the origin
     fs.write-file-sync path.join(service-origin-dir, 'new_file'), 'content'
     child_process.exec-sync "git add --all", cwd: service-origin-dir
-    child_process.exec-sync "git status", cwd: service-origin-dir
```
Comment (charlierudolph) [inline exo-sync/features/steps/steps-given.ls:29]: This was unneeded.

File: exo-sync/features/support/world.ls [hunk 2]
```diff
@@ -38,9 +38,12 @@ World = !->
     fs.rename-sync path.join(service-dir),
                    service-origin-dir
     child_process.exec-sync "git init", cwd: service-origin-dir
+    fs.write-file-sync path.join(service-origin-dir, "README.md"), 'my service'
+    child_process.exec-sync "git add README.md", cwd: service-origin-dir
+    child_process.exec-sync "git commit -m 'initial commit'", cwd: service-origin-dir
 
     # make the origin directory the new origin of the service
-    child_process.exec-sync "git clone #{service-origin-dir}", cwd: @app-dir
+    child_process.exec-sync "git clone -q #{service-origin-dir}", cwd: @app-dir
```
Comment (charlierudolph) [inline exo-sync/features/support/world.ls:46]: By default: `git clone` output progress infomation to stderr which was appearing in the log

File: exo-sync/features/support/world.ls [hunk 3]
```diff
@@ -38,9 +38,12 @@ World = !->
     fs.rename-sync path.join(service-dir),
                    service-origin-dir
     child_process.exec-sync "git init", cwd: service-origin-dir
+    fs.write-file-sync path.join(service-origin-dir, "README.md"), 'my service'
+    child_process.exec-sync "git add README.md", cwd: service-origin-dir
+    child_process.exec-sync "git commit -m 'initial commit'", cwd: service-origin-dir
```
Comment (charlierudolph) [inline exo-sync/features/support/world.ls:43]: These lines were added to avoid the warning about cloning an empty respository

**Review by cheahuychou — APPROVED** [review]

**Review by hugoesthere — APPROVED** [review]
Thanks for doing this!

**Pull request merged**